mcts/src/main/java/org/motechproject/nms/mcts/domain/MctsImportAudit.java

```diff
@@ -11,7 +11,10 @@
 public class MctsImportAudit {
 
     @Field
-    private LocalDate importDate;
+    private LocalDate startImportDate;
```

Comment:
pretty dangerous to change schema names in a live production system. This also breaks back compat. I wouldn't recommend this unless there is no other way around it.